scripts/support/create_ge_images/test_only.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+#
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FI-Core project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+
+cd $(dirname $0)
+. create_template.sh
+# Parameters: name
+check_params $*
+
+DIR=$IMAGES_DIR/$1
+user=$(cat $DIR/dist_type)
+# Delete old VM if it exists
+nova delete $(cat $DIR/last_vm 2>/dev/null) >/dev/null 2>&1  || true
+rm -f $DIR/last_vm
+test_template $1 $user
```

Comment:
IMAGE_DIR is defined when invoked ". create_template.sh", is the directory where the files are.

"user" may be ubuntu, centos, debian. It is the name of the account with sudo account in each image. These accounts were not created by us, there are in the mainstream images but usually all of them uses as username the name of the distribution.

User is read from a file, where was saved in the last execution of the create script. In the create script this information was available, but not in the test_only it is not. The user variable may be deleted, it is provided to the test script as USER but the test script should be specific for the distribution and the value could be hardcoded.